docs/contrib/PlatformBringUp.md

```diff
@@ -0,0 +1,415 @@
+set_global_routing_layer_adjustment $::env(MIN_ROUTING_LAYER)-$::env(MAX_ROUTING_LAYER) 0.5
+
+set_routing_layers -signal $::env(MIN_ROUTING_LAYER)-$::env(MAX_ROUTING_LAYER)
+```
+
+
+The first command, set_global_routing_layer_adjustment, adjusts the routing resources of the design. This effectively reduces the number of routing tracks that the global router assumes to exist. By setting it to the value of 0.5, this reduced the routing resources of all routing layers to 50% which can help with congestion and reduce the challenges for detail routing. The second command, set_routing_layers, sets the minimum and maximum routing layers for signal nets by using the -signal option.
+
+More customization can be done to increase the efficiency of global and detail route. Refer to the [OpenROAD documentation](https://openroad.readthedocs.io/_/downloads/en/latest/pdf/).
+
+
+## Metal Tracks Configuration
+
+OpenROAD requires a metal track configuration file for use in floorplanning. For each metal layer, the x and y offset as well as the x and y pitch are defined. To find the pitch and offset for both x and y, refer to the tech LEF. Following is a generalized metal tracks configuration file with five metal tracks defined.
+
+
+```
+# make_tracks.tcl
+###############################
+make_tracks metal1 -x_offset 0.24 -x_pitch 0.82 -y_offset 0.24 -y_pitch 0.82
+make_tracks metal2 -x_offset 0.28 -x_pitch 0.82 -y_offset 0.28 -y_pitch 0.82
+make_tracks metal3 -x_offset 0.28 -x_pitch 0.82 -y_offset 0.28 -y_pitch 0.82
+make_tracks metal4 -x_offset 0.28 -x_pitch 0.82 -y_offset 0.28 -y_pitch 0.82
+make_tracks metal5 -x_offset 0.28 -x_pitch 0.82 -y_offset 0.28 -y_pitch 0.82
+```
+
+
+Refer to the tech file to determine the x and y pitch and the offset.
+
+
+## PDN Configuration
+
+PDN is a utility that simplifies adding a power grid into the floorplan. With specifications given in the PDN configuration file, like which layer to use, stripe width and spacing, the utility can generate the metal straps used for the power grid. To create and configure a power grid, refer to [OpenROAD documentation](https://github.com/The-OpenROAD-Project/OpenROAD/blob/f864efbae4ffd4cc533f0e22c57cc7ab89618c44/src/pdn/doc/PDN.md) on pdngen.
+
+
+## Tapcell Configuration
+
+The tapcell configuration file is used to insert tapcells and endcaps into the design. Refer to the [Tapcell][https://github.com/The-OpenROAD-Project/OpenROAD/blob/f864efbae4ffd4cc533f0e22c57cc7ab89618c44/src/tap/README.md] documentation on how to construct this file.
+
+
+
+## setRC Configuration
+
+
+setRC allows the user to define resistances and capacitances for layers and vias using the `set_layer_rc` command. There is also a command that allows you to set the resistance and capacitance of routing wires using the `set_wire_rc`. Following is a generic example of a setRC configuration file which sets the resistance and capacitance of five metal layers, four vias, one signal wire, and one clock wire.
```

Comment:
The instructions should probably mention that per-unit-length values are often available in the PDK user guide. They can also be extracted from the commercial enablement using a commercial tool.